department-of-veterans-affairs/abd-vro
# Notify Slack when deploy scripts run

## What was the problem?


No Slack notifications when a deploy script is run manually in their local env.

Associated tickets or Slack threads:
- #769

## How does this fix it?


Add a `notify-slack.sh` script and use it in `deploy-app,core,db,mq,redis.sh` scripts.

## How to test this PR

Run `scripts/deploy-X.sh dev` locally (where X is one of `app, core, db, mq, redis`) and check notifications in `#benefits-vro-devops`. Note: `deploy-core.sh` will delete the persistence layer and dependent services (i.e., Postgres, RabbitMQ, Redis).
